source/app/powertabeditor.cpp

```diff
@@ -260,14 +261,14 @@ void PowerTabEditor::openFile(QString filename)
     int validationResult = myDocumentManager->findDocument(path);
     if (validationResult > -1)
     {
-        qDebug() << "File: " << filename << " is already open";
+        Log::d("file: {} is already open", filename.toStdString());
```

Comment:
So I tried this out but I'm still getting compilation errors.  I'm assuming that I should include this (`app/log.h`) after including `util/log.h`:
```cc
template <>
struct std::formatter<QString> : std::formatter<std::string>
{
    auto format(const QString& qstr, std::format_context& ctx) const {
        return std::formatter<std::string>(qstr.toStdString(), ctx);
    }
};
```

This gives me a compilation error; I'm going to go over this this weekend if I have the time.

Reply:
Hmm, maybe it needs to be in the `fmt` namespace rather than `std` to work with the fmt library?

I'd also be fine having `app/log.h` include `util/log.h` so that code in the UI libraries just needs to include one header to get access to logging plus the Qt format conversions

Reply:
So I just realized that I basically forgot to rename the guard of the previous `app/log.h` in `util/log.h`.  This resulted into a few hours of insanity of why do these two headers flip each other off or on (haha). 

Got this to work!

Reply:
Great!